summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/batch/BatchedStore.scala

```diff
@@ -33,6 +33,18 @@ import cascading.flow.FlowDef
 import org.slf4j.LoggerFactory
 
 import StateWithError.{ getState, putState, fromEither }
+import com.twitter.scalding.serialization.macros.impl.BinaryOrdering
+
+private[batch] case class LTuple2[T, U](_1: T, _2: U) {
```

Comment:
Sure, any concerns about doing it. Needs a comment tho

In general should we add a set of these Tuples to algebird/bijection at some point maybe?